jsii-config is missing a dependency on jsii-spec

## :bug: Bug Report

```shell
❯ npx jsii-config
npx: installed 65 in 9s
Cannot find module 'jsii-spec
```

### General Information
* **JSII Version:** 0.20.7.

--- a/packages/jsii-config/package.json
+++ b/packages/jsii-config/package.json
@@ -27,11 +27,11 @@
     "eslint": "^6.6.0",
     "jest": "^24.9.0",
     "jest-expect-message": "^1.0.2",
-    "jsii-spec": "^0.20.7",
     "typescript": "~3.6.4"
   },
   "dependencies": {
     "inquirer": "^7.0.0",
+    "jsii-spec": "^0.20.7",
     "yargs": "^14.2.0"
   },
   "jest": {

